server/cache_utils.py

```
@@ -741,7 +821,10 @@ def run_model(model_spec, prompt=None, interactive=False, temperature=0.7,
             prompt = prompt or "Hello"
 
         print(f"Prompt: {prompt}\n")
-        os.system(f'python -m mlx_lm generate --model "{model_path}" --prompt "{prompt}"')
+        os.system(
```

Ruff (0.14.10)

Starting a process with a shell, possible injection detected

(S605)